pre-exec.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+###############################################################################
+# See documentation at https://github.com/htcondor/htcondor/tree/main/build/docker/services#providing-additional-configuration
+# Condor Signing Key (Required to create future tokens) is automatically created upon starting this container
+# Once you have the key, you should provide it so it stays the same forever.
+###############################################################################
+if [ "$CONDOR_SIGNING_KEY" ] ; then
+    echo "$CONDOR_SIGNING_KEY" > /etc/condor/passwords.d/POOL
+    chmod 600 /etc/condor/passwords.d/POOL
+fi
+
+###############################################################################
+# Give specific JWT tokens permissions for Running Workers, Schedd, And Jobs
+###############################################################################
+if [ "$UID_DOMAIN" ] ; then
+    echo "ALLOW_ADVERTISE_STARTD = \$(ALLOW_ADVERTISE_STARTD) kbase_workers@${UID_DOMAIN} nersc_workers@${UID_DOMAIN}" >> /etc/condor/condor_config.local
+    echo "ALLOW_WRITE = \$(ALLOW_WRITE) kbase_workers@${UID_DOMAIN} nersc_workers@${UID_DOMAIN} condor_pool@${UID_DOMAIN}" >> /etc/condor/condor_config.local
+fi
+
+if [ "$CONDOR_JWT_TOKEN" ] ; then
+     echo "$CONDOR_JWT_TOKEN" > /etc/condor/tokens.d/JWT
+     chmod 600 /etc/condor/tokens.d/JWT
+fi
+
+####################### HOST PATHS ############################################
+DIRS_TO_CREATE=$(condor_config_val DIRS_TO_CREATE)
```

Comment:
Can compare to https://github.com/kbase/condor/pull/31/files#r1109230734

Reply:
This adds the group to the condor user, and allows these files to be written by owner and group